AK/MemMem.h

```diff
@@ -115,7 +115,7 @@ static inline Optional<size_t> memmem_optional(const void* haystack, size_t hays
     }
 
     if (needle_length < 32) {
-        auto ptr = bitap_bitwise(haystack, haystack_length, needle, needle_length);
+        const auto* ptr = Detail::bitap_bitwise(haystack, haystack_length, needle, needle_length);
```

Comment:
:scream: west-const in Serenity in 2021?

Reply:
Whoops. I was just uh... making sure you'd been properly converted to east const. Fixed and pushed.